android/exponentview/src/main/java/host/exp/exponent/notifications/NotificationHelper.java

```diff
@@ -284,4 +285,48 @@ public static void scheduleLocalNotification(
       listener.onFailure(e);
     }
   }
+
+  private static String getJSONString(Object item) throws JSONException {
```

Comment:
can you make a new class for these under exponentview/host.exp.exponent.utils.JSONUtils